libs/bilge/src/Bilge/IO.hs

```diff
@@ -80,13 +91,77 @@ newtype HttpT m a = HttpT
                )
 
 class MonadHttp m where
-    getManager :: m Manager
+    handleRequest :: Request -> m (Response (Maybe LByteString))
+    handleRequest req = handleRequestWithCont req consumeBody
+    handleRequestWithCont :: Request -> (Response BodyReader -> IO a) -> m a
```

Comment:
We remove any knowledge of `Manager` in favour of the more general request handler function.